### ISSUE TITLE
Post-request processing

### ISSUE THREAD

**Issue by aleksousa** [tetratelabs/proxy-wasm-go-sdk#389]
Hello everyone. Based on the Envoy [documentation](https://www.envoyproxy.io/docs/envoy/latest/intro/life_of_a_request.html#post-request-processing), there are processes that run after request processing. Is there any way through wasm to execute something at this stage?

I tried to find a way to send metrics and information through a queue to a Wasm Service at the end of request processing, in an attempt to not impact the response time, but I couldn't find it. Is this possible?

Thank you in advance.

**Comment by jcchavezs** [comment]
Did you try https://github.com/tetratelabs/proxy-wasm-go-sdk/blob/main/proxywasm/types/context.go#L161C2-L161C18 ?

**Comment by aleksousa** [comment]
It doesn't have the behavior I was looking for, at least not in the test I performed. I conducted a test by creating the 'OnHttpStreamDone' function in my plugin and added a loop inside it with a log to delay its execution and check if I would receive the response before the log was displayed. The response was only received after its execution.

My idea was actually to see if there is any way to fully respond to the request for the consumer of the API, and only then execute something.
